Samples/Forms/Example.Forms.UWP/Example.Forms.UWP.csproj

```diff
@@ -20,29 +20,6 @@
     <PackageCertificateKeyFile>Example.Forms.UWP_TemporaryKey.pfx</PackageCertificateKeyFile>
     <ReleaseVersion>2.5</ReleaseVersion>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
```

Comment:
Do we need these configs for people not on ARM cpus? :)

Reply:
There is no more x86. It's x64 by now 😉